python/dataprep.py

```diff
@@ -25,42 +25,83 @@
 METADATA_LIST = os.getenv('METADATA_LIST').split()
 
 
-# **** RESHAPE data long -> wide by taxon *******
-# ***********************************************
+def load_labelled(SINCE_THRESHOLD, level='level2'):
+    if level=='agnostic' or level=='level1':
+        dataframe = pd.read_csv(
+        os.path.join(DATADIR, 'labelled.csv.gz'),
+        dtype=object,
+        compression='gzip'
+         )
+    else:
+        dataframe = pd.read_csv(
+        os.path.join(DATADIR, 'labelled_level2.csv.gz'),
+        dtype=object,
+        compression='gzip'
+    )
 
-# reshape to wide per taxon and keep the combined text so indexing is consistent when splitting X from Y
+    if level=='level2':
+        # Create World taxon in case any items not identified
+        # through doc type in clean_content are still present
+        dataframe.loc[dataframe['level1taxon'] == 'World', 'level2taxon'] = 'world_level1'
 
+    if level=='level1' or level=='level2':
```

Comment:
This could also be written as `level in ('level1', 'level2')`, which I think is a bit neater.

Reply:
Thanks - am making some further cahnges to this function for the next experiment so will add these style improvements then